harrisonDir/harrisonHomework.txt

```diff
@@ -0,0 +1,113 @@
+PART 1:
+
+a) true, because == treats strings and numbers as being equal
+
+b) false, because === does NOT treat strings and numbers as being equal (i.e. === cares about type)
+
+c) true, for the same reason as a)
+
+d) false, because while x == "x" is true, x == "x " is false
+
```

Comment:
This will return true, though part of the confusion in this question is that there is not actually a space in this equation. '' is not the same as ' '. So what we are essentially comparing at the end of this is x == x.